src/PermissionRegistrar.php

```diff
@@ -76,11 +78,14 @@ public function initializeCache()
         self::$pivotRole = config('permission.column_names.role_pivot_key') ?: 'role_id';
         self::$pivotPermission = config('permission.column_names.permission_pivot_key') ?: 'permission_id';
 
-        $this->cache = $this->getCacheStoreFromConfig();
+        $this->cache = $this->getCacheRepository();
     }
 
-    protected function getCacheStoreFromConfig(): \Illuminate\Contracts\Cache\Repository
+    public function getCacheRepository(): Repository
```

Comment:
Renaming this function seems like unnecessarily risky. Why not just change `protected` to `public` so this won't break other people's code?

Reply:
@freekmurze done,
I added a specific getter for that, not breaking changes